steps/cadence-pegasus-fill/configure.yml

```diff
@@ -0,0 +1,55 @@
+#=========================================================================
+# Cadence Pegasus Fill
+#=========================================================================
+# Author : James Thomas
+# Date   : November 2, 2020
+#
+
+name: cadence-pegasus-fill
+
+#-------------------------------------------------------------------------
+# Inputs and Outputs
+#-------------------------------------------------------------------------
+
+inputs:
+  - design.gds
+  - adk
+
+outputs:
+  - fill.gds
+
+#-------------------------------------------------------------------------
+# Commands
+#-------------------------------------------------------------------------
+
+commands:
+  # Generate the dummy fill GDS
+  - pegasus -drc -fill inputs/adk/{fill_rule_deck} -gds design.gds -top_cell {design_name} -dp 8
+  - mkdir -p outputs && cd outputs
+  - ln -sf ../{fill_gds} fill.gds
+
+
+#-------------------------------------------------------------------------
+# Parameters
+#-------------------------------------------------------------------------
+
+parameters:
+  design_name: undefined
+  # Use the rule deck "inputs/adk/${fill_rule_deck}"
+  fill_rule_deck: pegasus-fill.rul
```

Comment:
Can we rename it as ".rule" to be consistent with the other rule decks?

Reply:
The convention with Pegasus is to use the "rul" suffix (some of the GUIs only open "rul" files).

Reply:
Got it.